packages/components/src/system-message/system-message.tsx

```diff
@@ -63,7 +63,14 @@ const SystemMessage = (props: Props) => {
           <DeletedMessageContent
             timestamp={timestamp}
             text={props.text}
-            action={props.action}
+            action={
+              props.action
+                ? {
+                    label: props.action.label,
+                    onPress: props.action.onClick,
```

Comment:
@jkbktl I've noticed the pipeline has been failing. Please, could you let me know if this fix is sufficient?

Reply:
I'll check, but I don't think so, there is issue with discriminated union type 🤔

Reply:
Thanks, btw I removed the arg from `renderMessage` and used the `type` from outer scope, letting TS to infer the correct type. This line mainly about setting `onPress` to `onClick`.

Reply:
So I obviously forgot to change onClick here, it should be onPress in Prop Types, same as it's inside DeletedMessageContent, can you please fix it for me here?

And yes, it seems like it works with your solution, thank you!

Reply:
Fix at https://github.com/status-im/status-web/pull/372/commits/e3db2ff895d45c5d140a271d0f46e6bd6fe0f151.